DigitalOffice/Services/CompanyService/Repositories/CompanyRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Linq;
+
+namespace LT.DigitalOffice.CompanyService.Repositories
+{
+    public class CompanyRepository : ICompanyRepository
```

Comment:
UpdateCompany method just changes object's properties, not the actual data

Reply:
But EF tracks changes by default, doesn't it?

Reply:
you need to check it. But there is no test that does this in pull request.

Reply:
What do you suggest instead of Update?